webapp/src/components/calendar/fullCalendar.tsx

```diff
@@ -0,0 +1,209 @@
+            if (dateDisplayProperty && dateDisplayProperty?.type === 'updatedTime') {
+                dateFrom = new Date(card.updateAt || 0)
+                dateTo = new Date(card.updateAt || 0)
+            } else if (dateDisplayProperty && dateDisplayProperty?.type !== 'createdTime') {
+                const dateProperty = createDatePropertyFromString(card.fields.properties[dateDisplayProperty.id || ''] as string)
+                if (!dateProperty.from) {
+                    return []
+                }
+
+                // date properties are stored as 12 pm UTC, convert to 12 am (00) UTC for calendar
+                dateFrom = dateProperty.from ? new Date(dateProperty.from + (dateProperty.includeTime ? 0 : timeZoneOffset(dateProperty.from))) : new Date()
+                dateFrom.setHours(0, 0, 0, 0)
+                const dateToNumber = dateProperty.to ? dateProperty.to + (dateProperty.includeTime ? 0 : timeZoneOffset(dateProperty.to)) : dateFrom.getTime()
+                dateTo = new Date(dateToNumber + oneDay) // Add one day.
+                dateTo.setHours(0, 0, 0, 0)
+            }
+            return [{
+                id: card.id,
+                title: card.title,
+                extendedProps: {icon: card.fields.icon},
+                properties: card.fields.properties,
+                allDay: true,
+                start: dateFrom,
+                end: dateTo,
+            }]
+        })
+    ), [cards])
+
+    const renderEventContent = (eventProps: EventContentArg): JSX.Element|null => {
+        const {event} = eventProps
+        return (
+            <div>
+                <div className='octo-icontitle'>
+                    { event.extendedProps.icon ? <div className='octo-icon'>{event.extendedProps.icon}</div> : undefined }
+                    <div
+                        className='fc-event-title'
+                        key='__title'
+                    >{event.title || intl.formatMessage({id: 'KanbanCard.untitled', defaultMessage: 'Untitled'})}</div>
+                </div>
+                {visiblePropertyTemplates.map((template) => (
+                    <Tooltip
+                        key={template.id}
+                        title={template.name}
+                    >
+                        <PropertyValueElement
+                            board={board}
+                            readOnly={true}
+                            card={cards.find((o) => o.id === event.id) || cards[0]}
+                            contents={[]}
+                            comments={[]}
+                            propertyTemplate={template}
+                            showEmptyPlaceholder={false}
+                        />
+                    </Tooltip>
+                ))}
+            </div>
+        )
+    }
+
+    const eventClick = useCallback((eventProps: EventClickArg) => {
+        const {event} = eventProps
+        props.showCard(event.id)
+    }, [])
```

Comment:
`props.showCard` actually can change, so I would add it here in the  requirements